docs/api/api-methods.md

```diff
@@ -0,0 +1,189 @@
+# API Methods
+All API methods calls except for token generation are secured via [OAuth2](oauth.md).
+You need to provide token via `Authorization` header to access requested data like this: ```Authorization: "Bearer eyJ0eXAiOiJKV...s3SKg"```
```

Comment:
```suggestion
You need to provide token via `Authorization` header to access requested data like this: ```Authorization: Bearer eyJ0eXAiOiJKV...s3SKg```
```

If I'm not mistaken there must not be `"`

Reply:
Well I have copied this from headers of some testing communication so I find it right

Reply:
No, it doesn't work. I just run HTTP
```http
GET /api/v1/products
Authorization: "Bearer eyJ0eX....ec8Q7UEz6g"
cache-control: no-cache
Postman-Token: 98fb37b3-c8cc-4161-965d-8d8b19fd3d7f
User-Agent: PostmanRuntime/7.6.0
Accept: */*
Host: 127.0.0.1:8000
cookie: PHPSESSID=2b240a9bece386b474edbf2e82426426
accept-encoding: gzip, deflate


```

And server responded `The resource server rejected the request.`

So no, just because a tool formats the header in this fashion doesn't mean it is right

Reply:
Weird but ok, I have removed those characters